tasks/internal/process-file-updates-task/README.md

```diff
@@ -16,6 +16,9 @@ replacements to a yaml file that already exists. It will attempt to create a Mer
 | tempDir                        | temp dir for cloning and updates                                                                                                                                                         | Yes      | /tmp/$(context.taskRun.uid)/file-updates |
 | internalRequestPipelineRunName | name of the PipelineRun that called this task                                                                                                                                            | No       | -                                        |
 
+## Changes in 0.1.2
+* Add idempotent changes with some fix
```

Comment:
I think the description here can be improved with more details about the fix.

Reply:
in time - I saw the commit message, you could use the part regarding the fix, adapting it to fit here.

Reply:
Updated. Thanks